charts/sysdig/templates/clusterrole-node-analyzer.yaml

```diff
@@ -0,0 +1,85 @@
+{{- if .Values.rbac.create }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "sysdig.fullname" .}}-node-analyzer
+  labels:
+{{ include "sysdig.labels" . | indent 4 }}
+rules:
```

Comment:
@nkraemer-sysdig these changes will not make new RuntimeScanner work properly.
These are the minimum permissions needed for it:
```
- apiGroups: ["", "apps", "batch", "extensions"]
  resources:
    - "namespaces"
    - "deployments"
    - "replicasets"
    - "daemonsets"
    - "statefulsets"
    - "pods"
    - "cronjobs"
    - "jobs"
  verbs: ["get", "list", "watch"]
```

Reply:
@ProvoK Permissions have been added